apps/jupyter/extensions/notebook-runner/src/__tests__/task-detector.spec.ts

```diff
@@ -46,10 +46,11 @@ describe("detectTaskFormat", () => {
 
       const blob = await mockZip.generateAsync({ type: "blob" });
 
-      await expectErrorWithDetails(
-        () => detectTaskFormat(blob),
-        UnsupportedTaskFormatError,
+      const error = await expectError<UnsupportedTaskFormatError>(() =>
+        detectTaskFormat(blob),
       );
+
+      expect(error).toBeInstanceOf(UnsupportedTaskFormatError);
```

Comment:
Just wondering whether you may want to supply this as a parameter to the `expectError` function. Then there would be no need to repeat this + no need to pass the generic type argument